components/Players/PlayerManager.vue

```diff
@@ -195,6 +203,34 @@ export default Vue.extend({
         default:
           return this.getCurrentItem.Name;
       }
+    },
+    timeLeft(): number {
+      return parseInt((this.mediaDuration - this.getCurrentTime).toFixed());
+    },
+    mediaDuration(): number {
```

Comment:
This should probably be a getter on the playbackManager store instead, since it's something we're bound to re-use (In fact, we already have something similar for the duration in the music controls, only with added formatting).

Reply:
Sure, would it be best to return it in milliseconds or ticks?